app/i18n/en/feedback.php

```diff
@@ -47,6 +48,7 @@
 		'no_access' => 'You have no access on %s',
 		'not_enabled' => '%s is not enabled',
 		'not_found' => '%s does not exist',
+		'removed' => '%s deleted',
```

Comment:
Remove might be better than delete? But either way since they're new I'd keep the key and the text the same.

Reply:
Sure, I can change that.
The keys are not new, they've never been translated.

Reply:
done